packages/html/src/HTMLTooltip.ts

```diff
@@ -66,6 +100,10 @@ export class HTMLTooltip extends HTMLWidget {
         this.updateTooltipPosition();
     }
 
+    onShowContent(node) {
+        
+    }
```

Comment:
What does this event do?

Reply:
It's a hook for shown content (aka a callback that runs when the tooltip is displayed via _contentNode rather than html). I added a gallery sample where I use it to render a Pie chart in the tooltip:
demos/gallery/samples/misc/React Timeline Tooltip Content.js